tests/virt_autotest/guest_installation_run.pm

```diff
@@ -86,6 +86,19 @@ sub run {
         $upload_guest_assets_flag = 'yes';
     }
 
+    if (check_var('BOOT_64KB_PAGE_SIZE', '64kps')) {
```

Comment:
Personally I think 64kpg is better than 64kps

Reply:
How about KERNEL_64KB_PAGE_SIZE ? It represents the whole scenario instead of just "boot".

Reply:
Done